avm/src/daemon/linux.rs

```diff
@@ -65,9 +65,8 @@ async fn post_v1(endpoint: &str, body: Value) -> String {
   }
 }
 
-async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, alan_version: &str, factor: &str) -> String {
+async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, factor: &str) -> String {
   let scale_body = json!({
-    "alanVersion": alan_version,
```

Comment:
why are we removing this part? I feel like it's still important information, even if it's basically just a sanity check between all nodes

Reply:
Oh this was never really necessary since it is readily available via DNS for the recipient and all nodes in the cluster

Reply:
Updated the PR description too